src/core/thread/mle.cpp

```diff
@@ -3621,8 +3625,9 @@ void Mle::HandleChildUpdateResponse(RxInfo &aRxInfo)
 
 #if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
         // CSL Accuracy
-        if (Tlv::FindTlv(aRxInfo.mMessage, clockAccuracy) != kErrorNone)
```

Comment:
Adding a note here: I think there was a bug here:
- If `FindTlv()` was not successful then we could use the `clockAccuracy` content.
- If the TLV was found, then we would skip and not update `Mac`.

I guess impact of this is small since we expect CSL clock accuracy values to be exchanged during initial attach in "Parent Response" and won't likely change afterwards (which then require Child Update exchange).